cli/azd/resources/scaffold/templates/resources.bicept

```diff
@@ -119,13 +128,53 @@ module postgreServer 'br/public:avm/res/db-for-postgre-sql/flexible-server:0.1.4
     ]
     databases: [
       {
-        name: databaseName
+        name: postgreSqlDatabaseName
+      }
+    ]
+    location: location
+    {{- if (and .DbPostgres .DbPostgres.AuthUsingManagedIdentity) }}
+    roleAssignments: [
+      {
+        principalId: connectionCreatorIdentity.outputs.principalId
+        principalType: 'ServicePrincipal'
+        roleDefinitionIdOrName: 'b24988ac-6180-42a0-ab88-20f7382dd24c'
       }
     ]
+    {{- end}}
+  }
+}
+{{- end}}
+{{- if (or (and .DbMySql .DbMySql.AuthUsingManagedIdentity) (and .DbPostgres .DbPostgres.AuthUsingManagedIdentity))}}
+
+module connectionCreatorIdentity 'br/public:avm/res/managed-identity/user-assigned-identity:0.2.1' = {
+  name: 'connectionCreatorIdentity'
+  params: {
+    name: '${abbrs.managedIdentityUserAssignedIdentities}cci-${resourceToken}'
+    location: location
+  }
+}
+{{- end}}
+{{- if (and .DbPostgres .DbPostgres.AuthUsingManagedIdentity) }}
+{{- range .Services}}
+module {{bicepName .Name}}CreateConnectionToPostgreSql 'br/public:avm/res/resources/deployment-script:0.4.0' = {
+  name: '{{bicepName .Name}}CreateConnectionToPostgreSql'
+  params: {
+    kind: 'AzureCLI'
+    name: '${abbrs.deploymentScript}{{bicepName .Name}}-connection-to-pg-${resourceToken}'
+    azCliVersion: '2.63.0'
     location: location
+    managedIdentities: {
+      userAssignedResourcesIds: [
+        connectionCreatorIdentity.outputs.resourceId
+      ]
+    }
+    runOnce: false
```

Comment:
This is a great parameter.

Reply:
What does this mean?

Reply:
So we are still use deployment script to do the trick, does any database avm support passwordless yet?

Reply:
> What does this mean?

This parameter can solve this problem: https://github.com/azure-javaee/azure-dev/wiki/Knowledge-Sharing#solution

> ![image](https://github.com/user-attachments/assets/d00058e5-d9f8-48a3-b21c-becd301a8d6c)

Reply:
> So we are still use deployment script to do the trick, does any database avm support passwordless yet?

We are still use deployment script to do the trick